### PR TITLE
Fix filelog README.md

### PR DESCRIPTION
**Pull request by tigrannajaryan**
Fix the README.md to reflect the changes that were recently done to filelog receiver.

### CODE REVIEW

File: receiver/filelogreceiver/README.md [hunk 1]
```diff
@@ -1,23 +1,17 @@
 # README Status
 This readme is out of date and will be updated soon
 
-# Stanza Receiver
+# Filelog Receiver
 
-Tails and parses logs from a wide variety of sources using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
+Tails and parses logs from a wide files using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
```
Comment (anuraaga) [inline receiver/filelogreceiver/README.md:6]:
```suggestion
Tails and parses logs from files using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
```

**Review by anuraaga — APPROVED** [review]

**Review by djaglowski — APPROVED** [review]
Thanks for picking this up @tigrannajaryan.

**Comment by djaglowski** [comment]
I initially thought this would address #2335, but I see some more changes I would like to make. I'll make a follow up PR today.

File: receiver/filelogreceiver/README.md (hunk 1 above)
Comment (tigrannajaryan) [reply receiver/filelogreceiver/README.md:6]: Done here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2373

**Pull request merged**